appsembler/LingoX
# Import and refactor LingoX code from the Edraak repo

Closes #7.

### How to Test
 - `vagrant ssh` and activate the `edxapp` environment
 - `$ pip install -e git+https://github.com/appsembler/LingoX.git@omar/migrate#egg=lingox`
 - Add `lingox` to the `ADDL_INSTALLED_APPS`.
 - Set `LANGUAGE_CODE` to `ar`
 - Reload the LMS server
 - Open a new incognito window on `http://localhost:8000/`, you should see an Arabic interface

I'm planning to do minor improvements including docs and Sites (aka microsites) support. More on the issues:

 - https://github.com/appsembler/LingoX/issues
